packages/patternfly-4/src/components/content/icons.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
-import { Gallery, Form, TextInput } from '@patternfly/react-core';
+import { Gallery, Form, TextInput, Button, Title, EmptyState, EmptyStateVariant, EmptyStateIcon, EmptyStateBody } from '@patternfly/react-core';
 import * as icons from '@patternfly/react-icons';
 import IconCard from './iconCard';
 import paramCase from 'param-case';
 import coreIcons from '../../../_repos/core/src/icons/definitions/pf-icons.json';
 /** @jsx jsx */
 import { css, jsx } from '@emotion/core';
 import './icons.scss';
+import { SearchIcon } from '@patternfly/react-icons';
```

Comment:
not needed since we already import it on line 3. I think you can do 
```
<EmptyStateIcon icon={icons.SearchIcon}/>
```

Reply:
updated @jschuler